core/src/test/scala/org/apache/spark/sql/delta/optimize/DeltaReorgSuite.scala

```diff
@@ -108,6 +123,31 @@ class DeltaReorgSuite extends QueryTest
       val (addFiles2, _) = getFileActionsInLastVersion(log)
       assert(addFiles2.size === 2)
       assert(addFiles2.forall(_.deletionVector === null))
+
+      // Verify commit history and operation metrics
+      checkOpHistory(
+        tablePath = path.toString,
+        expOpParams = Map("applyPurge" -> "true", "predicate" -> "[\"'part IN (0,2)\"]"),
+        numFilesRemoved = 2,
+        numFilesAdded = 2)
     }
   }
+
+  private def checkOpHistory(
+      tablePath: String,
+      expOpParams: Map[String, String],
+      numFilesRemoved: Long,
+      numFilesAdded: Long): Unit = {
+    val (opName, opParams, opMetrics) = DeltaTable.forPath(tablePath)
+      .history(1)
+      .select("operation", "operationParameters", "operationMetrics")
+      .as[(String, Map[String, String], Map[String, String])]
+      .head()
+    assert(opName === "REORG")
```

Comment:
Do we check for opName in the optimize suites?

Reply:
We are [not](https://github.com/delta-io/delta/blob/master/core/src/test/scala/org/apache/spark/sql/delta/optimize/OptimizeMetricsSuite.scala). Will followup with a separate PR.

Reply:
https://github.com/delta-io/delta/issues/1756

Reply:
https://github.com/delta-io/delta/pull/1758